core/src/test/scala/unit/kafka/cluster/AbstractPartitionTest.scala

```diff
@@ -49,7 +49,7 @@ class AbstractPartitionTest {
   var logDir1: File = _
   var logDir2: File = _
   var logManager: LogManager = _
-  var alterIsrManager: MockAlterIsrManager = _
+  var alterIsrManager: MockAlterPartitionManager = _
   var isrChangeListener: MockIsrChangeListener = _
```

Comment:
Should we also change IsrChangeListener to PartitionChangeListener?

Reply:
Yeah, makes sense.